frappe/public/js/frappe/router.js

```diff
@@ -38,16 +38,17 @@ $("body").on("click", "a", function (e) {
 		return false;
 	};
 
-	const href = e.currentTarget.getAttribute("href");
+	const targetElement = e.currentTarget;
```

Comment:
let's use snake_case instead of camelCase. Since we use it everywhere in our code base.

Reply:
Done. FC Habit :)